CONTRIBUTING.md

```diff
@@ -241,10 +247,10 @@ Execution failed for task ':arrow-core:apiCheck'.
   +++ /Users/john/projects/arrow/arrow-libs/core/arrow-core/build/api/arrow-core-retrofit.api
  ```
 
-To make the check pass you need to run:
+To make the check pass you need to run the `apiDump` task in the subproject which API you modified (here `arrow-core` as example):
 
 ```bash
-./gradlew apiDump
+./gradlew :arrow-core:apiDump
```

Comment:
Is it not easier to just run `apiDump` in all projects?

Reply:
Maybe you're right... I just tested on a project without API changes.
The initial run of `./gradlew apiDump` took 2 min 13 s on my MacBook.
All the subsequent runs take around 50-55 s, regardless if I run the task on all projects with `./gradlew apiDump` or just one subproject with `./gradlew :arrow-core-retrofit:apiDump`. This is because Gradle recognizes the task as up-to-date:
```
> Task :arrow-core-retrofit:compileKotlin UP-TO-DATE
> Task :arrow-core-retrofit:compileJava NO-SOURCE
> Task :arrow-core-retrofit:classes UP-TO-DATE
> Task :arrow-core-retrofit:apiBuild UP-TO-DATE
> Task :arrow-core-retrofit:apiDump UP-TO-DATE
```
So it's probably better to remove this remark 👍